packages/ui/components/dialog/Dialog.tsx

```diff
@@ -190,6 +190,7 @@ export function DialogFooter(props: DialogFooterProps) {
       <div
         className={classNames(
           "flex justify-end space-x-2 pb-4 pt-4 rtl:space-x-reverse",
+          "bg-default -ml-1 -mr-1",
```

Comment:
It's works, but definitely not pixel perfect. My question is, for one dialog or a simple issue, why do other dialogs suffer?

![Screenshot 2024-08-16 193625](https://github.com/user-attachments/assets/38fc2ced-101e-433c-bb2e-11b5ae7d80de)

People often don't provide as much description when creating an event, which causes the editor to go off-screen. I suggest adding an optional max-height property to the editor to ensure it doesn't go off-screen.